integration-tests/main/src/test/java/io/quarkus/it/main/MockTestCase.java

```diff
@@ -0,0 +1,60 @@
+package io.quarkus.it.main;
+
+import javax.enterprise.inject.spi.CDI;
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.it.mock.MockableBean1;
+import io.quarkus.it.mock.MockableBean2;
+import io.quarkus.test.junit.MockSupport;
+import io.quarkus.test.junit.QuarkusMock;
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class MockTestCase {
+
+    protected static final String HI = "Hi ";
+    protected static final String BONJOUR = "Bonjour ";
+
+    @Inject
+    MockableBean1 mockableBean1;
+
+    @Inject
+    MockableBean2 mockableBean2;
+
+    @BeforeAll
+    public static void setup() {
+        QuarkusMock.installMock(CDI.current().select(MockableBean1.class).get(), new HiGreeter());
```

Comment:
You could just use `QuarkusMock.installMock(mockableBean1, new HiGreeter())`, right?

Reply:
It's a static method

Reply:
I see. That's not very nice though. How about specifying the required type/qualifiers, i.e. in this case something like `QuarkusMock.installMock(MockableBean1.class, new HiGreeter())`?

Reply:
And do the lookup under the hood.